SnapMD.VirtualCare.ApiModels/OnDemandIntakeRequest.cs

```diff
@@ -10,18 +10,43 @@
 //    limitations under the License.
 
 using System.Collections.Generic;
+using SnapMD.VirtualCare.Sdk.Models;
 
-namespace SnapMD.VirtualCare.Sdk.Models
+namespace SnapMD.VirtualCare.ApiModels
 {
-    public class OnDemandConsultationRequest
+    /// <summary>
+    /// 
+    /// </summary>
+    public class OnDemandIntakeRequest
     {
-        public OnDemandConsultationRequest()
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OnDemandIntakeRequest"/> class.
```

Comment:
@jay-lee00 because of the way the documentation compilers format the text (Sandcastle, Swagger, etc.), there shouldn't be periods in the Summary section.